tests/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ macro (build_and_run arg)
 	target_link_libraries(Sqlpp11MySQL${arg} sqlpp-mysql)
 	target_link_libraries(Sqlpp11MySQL${arg} Threads::Threads)
 	target_link_libraries(Sqlpp11MySQL${arg} "${MYSQL_LIBRARIES}")
+	target_link_libraries(Sqlpp11MySQL${arg} sqlpp11::sqlpp11)
```

Comment:
This line isn't necessary anymore as sqlpp11 is (or rather should be) already part of the public interface

Reply:
For some reason it didn't compile before, now after the proposed updates it compiles normally.